apiserver/gui.go

```diff
@@ -246,8 +245,7 @@ func (h *guiHandler) serveStatic(w http.ResponseWriter, req *http.Request) {
 	staticDir := filepath.Join(h.rootDir, "static")
 	logger.Tracef("serving Juju Dashboard static files from %q", staticDir)
 	fs := http.FileServer(http.Dir(staticDir))
-	prefix := path.Join(h.basePath, "static")
-	http.StripPrefix(prefix, fs).ServeHTTP(w, req)
+	http.StripPrefix("/static/", fs).ServeHTTP(w, req)
```

Comment:
Since we set the serving of the static files from the root, we needed to update the stripping of the prefix for the file server.